test/git_SUITE.erl

```diff
@@ -84,7 +84,7 @@ ignore_deleted_files(Config) ->
     PrivDir = proplists:get_value(priv_dir, Config),
     ok = file:set_cwd(PrivDir), %ct will set cwd of each test to its log_dir
     _ = os:cmd("git init ."),
-    ok = file:write_file("test", <<"ingore">>, [append]),
+    ok = file:write_file("test", <<"ignore">>, [append]),
```

Comment:
This change likely broke some tests. Do you mind fixing them?

Reply:
Done.